server/vulnerabilities/nvd/cpe.go

```diff
@@ -152,11 +152,11 @@ func cpeGeneralSearchQuery(software *fleet.Software) (string, []interface{}, err
 
 	// 4 - Try vendor/product from bundle identifier, like tld.vendor.product
 	bundleParts := strings.Split(software.BundleIdentifier, ".")
-	if len(bundleParts) >= 3 {
+	if len(bundleParts) == 3 {
```

Comment:
If we encounter software with a long bundle identifier that we need to match, we can add it later. I scanned a list of software on my machine and didn't see anything interesting with a long bundle identifier that we'd want to match.